packages/components/src/components/table-stateless/component.tsx

```diff
@@ -720,35 +727,42 @@ export class KolTableStateless implements TableStatelessAPI {
 									this.state._headerCells.horizontal.map((cols, rowIndex) => (
 										<tr key={`thead-${rowIndex}`}>
 											{this.state._selection && this.renderHeadingSelectionCell()}
-											{cols.map((cell, colIndex) => {
-												if (cell.asTd === true) {
-													return (
-														<td
-															key={`thead-${rowIndex}-${colIndex}-${cell.label}`}
-															class={{
-																[cell.textAlign as string]: typeof cell.textAlign === 'string' && cell.textAlign.length > 0,
-															}}
-															colSpan={cell.colSpan}
-															rowSpan={cell.rowSpan}
-															style={{
-																textAlign: cell.textAlign,
-																width: cell.width,
-															}}
-															ref={
-																typeof cell.render === 'function'
-																	? (el) => {
-																			this.cellRender(cell, el);
-																		}
-																	: undefined
-															}
-														>
-															{typeof cell.render !== 'function' ? cell.label : ''}
-														</td>
-													);
-												} else {
-													return this.renderHeadingCell(cell, rowIndex, colIndex, false);
-												}
-											})}
+											{rowIndex === 0 &&
```

Comment:
Die Render-Methode ist ohnehin schon sehr komplex, daher würde ich vorschlagen, diese Expression analog der Spacer-Zellen in eine neue Methode auszulagern (`renderPlaceholderCell` z.B.).